docs/conf.py

```diff
@@ -1 +1,4 @@
 from crate.theme.rtd.conf.crash import *
+
+site_url = 'https://crate.io/docs/clients/crash/en/latest/'
+extensions = ['sphinx_sitemap']
```

Comment:
we usually put a new line at the end of the files.
in sublime text you could add this to your settings: `	"ensure_newline_at_eof_on_save": true,
`
otherwise ack